plans/burdon/plugin-refactor/task_plan.md

```
@@ -0,0 +1,44 @@
+
+### Phase 4: Audit other plugins (plugin-table, plugin-sheet, etc.)
+- [ ] Apply same pattern across other plugins
+- **Status:** pending
+
+### Phase 5: Lint & PR
+- [ ] pnpm -w pre-ci
+- [ ] Submit PR
+- **Status:** pending
+
+## Decisions Made
+| Decision | Rationale |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)